lua/lspconfig/configs/autohotkey_lsp.lua

```diff
@@ -0,0 +1,67 @@
+-- NOTE: AutoHotkey is used only on windows
+local util = require 'lspconfig.util'
+
+local function get_autohotkey_path()
+  local handle = io.popen 'where.exe autohotkey.exe'
```

Comment:
doesn't hurt to be cross-platform where possible, even if autohotkey is only relevant on Windows. It also can improve functionality.

```suggestion
  local handle = vim.fn.exepath('autohotkey.exe')
```